packages/core/src/telemetry/clearcut-logger/event-metadata-key.ts

```diff
@@ -234,4 +234,26 @@ export enum EventMetadataKey {
 
   // Logs the number of tokens after context window compression.
   GEMINI_CLI_COMPRESSION_TOKENS_AFTER = 61,
+
+  // ==========================================================================
+  // Research Event Keys
+  // ===========================================================================
+
+  // Logs the research opt-in status (true/false)
+  GEMINI_CLI_RESEARCH_OPT_IN_STATUS = 64,
```

Comment:
As you've discovered, this file is a bit of a pain in that it needs to be kept in sync vs an internal system.  You'll need a googler to send that change internally for you.  I'm happy to do this once we get thru another review cycle. 

FYI: We might have to change these numbers, but this looks fine for the moment.